pyrometheus/codegen/python.py

```diff
@@ -335,23 +368,24 @@ def get_species_enthalpies_rt(self, temperature):
             % endfor
             ])
 
-    def get_species_entropies_r(self, temperature):
+    def get_species_entropies_r(self, pressure, temperature):
         return self._pyro_make_array([
             % for sp in sol.species():
-            ${cgm(ce.poly_to_entropy_expr(sp.thermo, "temperature"))},
+            ${cgm(ce.poly_to_entropy_expr(sp.thermo, "temperature"))}
+            - self.usr_np.log(pressure/101325.0),
             % endfor
             ])
 
-    def get_species_gibbs_rt(self, temperature):
+    def get_species_gibbs_rt(self, pressure, temperature):
         h0_rt = self.get_species_enthalpies_rt(temperature)
-        s0_r = self.get_species_entropies_r(temperature)
+        s0_r = self.get_species_entropies_r(pressure, temperature)
         return h0_rt - s0_r
 
-    def get_equilibrium_constants(self, temperature):
+    def get_equilibrium_constants(self, pressure, temperature):
         rt = self.gas_constant * temperature
-        c0 = self.usr_np.log(self.one_atm / rt)
+        c0 = self.usr_np.log(pressure / rt)
```

Comment:
Does the `self.usr_np.log(pressure / rt)` along with the `np.log(x)` term in the mixture entropy not reduce to `self.usr_np.log(self.one_atm / rt)`? Somehow I think it is related to why we represented it in this way to begin with. I looked at the math of the `\sum_i x_i log(x_i)` term a long time ago for a couple of different reasons, though I can't remember for sure if it impacts this term.

Reply:
I don't get what you are meaning, exactly. Could you elaborate, please?